readthedocs/api/v3/views.py

```diff
@@ -177,11 +179,23 @@ def get_queryset(self):
         # This could be a class attribute and managed on the ``ProjectQuerySetMixin`` in
         # case we want to extend the ``prefetch_related`` to other views as
         # well.
-        return queryset.prefetch_related(
-            "related_projects",
-            "domains",
```

Comment:
These were adding extra queries and not being used, as django ignores prefetching if anything other than objects.all() is called.